packages/devtools/lib/src/service_manager.dart

```diff
@@ -226,6 +226,34 @@ class ServiceConnectionManager {
       isolateId: _isolateManager.selectedIsolate.id,
     );
   }
+
+  Future<int> getDisplayRefreshRate() async {
+    final flutterViewListResponse = await service.callServiceExtension(
+      registrations.flutterListViews,
+      isolateId: _isolateManager.selectedIsolate.id,
+    );
+    final List<dynamic> views =
+        flutterViewListResponse.json['views'].cast<Map<String, dynamic>>();
+
+    // Each isolate should only have one FlutterView.
+    final flutterView = views.firstWhere(
+      (view) => view['type'] == 'FlutterView',
+      orElse: () => null,
+    );
+
+    if (flutterView == null) {
+      print('No Flutter Views to query: ${flutterViewListResponse.json}');
```

Comment:
I wonder if we should have a way to log debug messages to the chrome devtools console, even if it's just a simple alias for `print`?

Reply:
If we're not able to find a Flutter View, should we return `null`? Return `0`? Return some default value, even if it may not be correct (`60`)?

Reply:
Could there be a race condition if we query this API too early (for example before the first frame has drawn)?
It could be safest to make this return an error if the data isn't available so we don't quietly return 60 if we have a timing bug.

Reply:
added a Logger class to handle DevTools logging to console.

if `flutterView == null`, we will throw an Error.